.dockerignore

```diff
@@ -1,7 +1,52 @@
-README.md
-test/
-.git/
-.github/
-.credo.exs
-.formatter.exs
+# This file excludes paths from the Docker build context.
```

Comment:
From `mix phx.gen.release --docker` with some added modifications on the bottom.